_includes/install-nav.html

```diff
@@ -61,6 +61,20 @@
     <div class="version-header">
       MoveIt 2.0
     </div>
+    <a href="/install-moveit2/binary/">
+      {% if page.url == "/install-moveit2/binary/" %}
+        <div class="row no-gutters">
+          <img src="/assets/install_page/current_page_left.png" class="current-page-image-left">
+          <img src="/assets/install_page/current_page_right.png" class="current-page-image-right">
+          <div class="font-current-page">
+      {% else %}
+        <div>
+          <div class="row font-other-page">
+      {% endif %}
+            Binary Install
```

Comment:
```suggestion
            Binary Install: Linux
```

For consistency with the link below.